meta/main.yml

```diff
@@ -1,14 +1,14 @@
 ---
 
 galaxy_info:
+  author: idealista
   company: Idealista S.A.U.
   description: Prometheus Consul exporter
-  min_ansible_version: 2.4.0.0
+  min_ansible_version: 2.8.8
```

Comment:
As we are testing with Ansible 2.9.9 I think it would be nice to specify that version here

Reply:
done